azure-iot-device/samples/advanced-hub-scenarios/send_message_over_websockets.py

```diff
@@ -0,0 +1,37 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import os
+import asyncio
+from azure.iot.device.aio import IoTHubDeviceClient
+
+
+async def main():
+    # Fetch the connection string from an enviornment variable
+    conn_str = os.getenv("IOTHUB_DEVICE_CONNECTION_STRING")
+
+    # Create instance of the device client using the connection string
+    device_client = IoTHubDeviceClient.create_from_connection_string(conn_str, websockets=True)
+
+    # Connect the device client.
+    await device_client.connect()
```

Comment:
This isn't necessary, remove (I know some samples have it, but we're trying to avoid it)